azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -124,8 +124,7 @@ public void selectSubscription(List<String> selectedSubscriptionIds) {
                 manager.runOnPooledThread(Preloader::load);
             }
         } else {
-            throw new AzureToolkitAuthenticationException("no subscriptions are selected, " +
-                    "make sure you have provided valid subscription list");
+            throw new AzureToolkitRuntimeException("the selected subscriptions are invalid", IAccountActions.SELECT_SUBS, IAccountActions.AUTHENTICATE);
```

Comment:
do we need an new action `RE_AUTHENTICATE`?

Reply:
not sure is re_authenticate is required